src/domain/post.js

```diff
@@ -35,6 +37,19 @@ export default class Post {
     return Post.fromDb(createdPost)
   }
 
+  async update() {
+    const updatedPost = await dbClient.post.update({
+      where: {
+        id: this.id
+      },
+      data: {
+        content: this.content,
+        edited: this.edited
```

Comment:
This will constantly overwrite the value with true, even if already true. Let's be smarter about this - only once do we need to update from false to true.

Reply:
```js
      data: {
        content: this.content,
        edited: this.edited || undefined
      }
```
> changed it to undefined if it's already true ,

As I understand this, the only way `data.edited` is undefined, is if `this.edited` is false? The problem I mentioned above is when `this.edited` is true - it will keep overwriting data.

Reply:
changed to && so now if it is true then it gonna go to the undefined also added user include statement when i fetch all the posts because it was needed for my front end  to work.